packages/image_picker/image_picker_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.3.1
```

Comment:
should be 2.4.0 as there are non-breaking changes to the public API, same thing in pubspec.yaml

Reply:
I've updated this as well as the version & changelog for web

Reply:
I think the web change needs to be in the other PR

Reply:
Since web directly `extends ImagePickerPlatform`, it's required to match the function signatures for `ImagePickerPlatform`

As a result, the automated web tests fail if it's not also changed to match. Granted, there's no changes to web beyond adding `bool forceFullMetadata = true,` to the function signatures.

Reply:
hmm. This is weird. I would think it is the other way around. Since the platform_interface change hasn't been published, the web shouldn't pick up the newest API, thus the override should report a compile error.

Reply:
@cpboyd Do you mind double checking again. The test for web should not fail because the web package is depending on an old version of platform_interface, where the method signature hasn't been changed.

Reply:
@cyanglaz I just pushed a clean commit without the web changes

Reply:
> Since web directly `extends ImagePickerPlatform`, it's required to match the function signatures for `ImagePickerPlatform`
> 
> As a result, the automated web tests fail if it's not also changed to match. Granted, there's no changes to web beyond adding `bool forceFullMetadata = true,` to the function signatures.

This is a description of why this is a breaking change, so I'm not clear on why this was landed. You cannot add parameters to methods to an interface without breaking all implementations.

You also cannot just ignore that something is a breaking change by also pushing a change in another package. I don't understand how this was approved with *unpublished* code changes to a separate package.

Reply:
> @cyanglaz I just pushed a clean commit without the web changes

Part of the problem here is that this did not actually happen :(